src/FieldTrait.php

```
@@ -174,37 +172,34 @@ public function fieldFillWysiwyg(string $field, string $value): void {
           }
-        } else {
-          throw new Exception('Could not find the element!');
-        }
-        ");
+          if (!element.ckeditorInstance) {
+            throw new Error('CKEditor instance not found for element with ID {$element_id}');
+          }
+          element.ckeditorInstance.setData("{$value}");
+        })();
+JS;
+    }
```

PHPMD (2.15.0)

The method fieldFillWysiwyg uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)